src/general/semidiscretization.jl

```diff
@@ -854,3 +854,13 @@ function check_configuration(system::TotalLagrangianSPHSystem, systems)
                             "`ContinuityDensity` is not yet supported for a `TotalLagrangianSPHSystem`"))
     end
 end
+
+function check_configuration(system::OpenBoundarySPHSystem, systems)
+    (; boundary_model, boundary_zone) = system
+
+    if boundary_model isa BoundaryModelLastiwka &&
+       first(typeof(boundary_zone).parameters) === BidirectionalFlow
+        throw(ArgumentError("`BoundaryModelLastiwka` needs a specific flow direction. " *
+                            "Please specify inflow and outflow."))
+    end
```

Comment:
So the indention for this model is just pipe and channel flows?
This should be clarified in the documentation.